examples/splunk-sink/example.auto.tfvars

```diff
@@ -0,0 +1 @@
+../example.auto.tfvars
```

Comment:
Opinion: In addition I think that we should name this `terraform.tfvars.sample` or `example.auto.tfvars.sample` - I'd like to drop in my own tfvars without having to delete or modify files owned by Git.

Make this change if you agree, resolve this comment otherwise.

Reply:
+1